paddle/api/CMakeLists.txt

```diff
@@ -92,7 +92,9 @@ add_custom_command(OUTPUT ${PROJ_ROOT}/paddle/py_paddle/_swig_paddle.so
 # TODO(yuyang18) : make wheel name calculated by cmake
 add_custom_target(python_api_wheel ALL DEPENDS ${PROJ_ROOT}/paddle/py_paddle/_swig_paddle.so)
 
-install(DIRECTORY ${PROJ_ROOT}/paddle/dist/ DESTINATION opt/paddle/share/wheels)
+install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/dist/
```

Comment:
Yeah. ${CMAKE_CURRENT_SOURCE_DIR} means the current source code directory.
${CMAKE_CURRENT_SOURCE_DIR} denotes the root directory of source code.